packages/frontend/src/stdlib/analyses/sql.tsx

```diff
@@ -0,0 +1,110 @@
+// import type { JSX } from "solid-js";
+
+import { DropdownMenu } from "@kobalte/core/dropdown-menu";
```

Comment:
FYI this has been removed in the meantime so your merge with `main` broke things. Have a look here for replacing this with `@corvu/popover`

https://github.com/ToposInstitute/CatColab/pull/826/files#diff-a078dd4015617a7e02c970ddeceaca6e0798c5cbe1bd0306511532c517eb11e8